### PR TITLE
Update docs, run gendocs in travis

### PR DESCRIPTION
**Pull request by thockin**
@jlowdermilk 

This is causing trouble in a totally unrelated PR

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ install:
   - ./hack/verify-gofmt.sh
   - ./hack/verify-boilerplate.sh
   - ./hack/verify-description.sh
+  - ./hack/verify-gendocs.sh
```
Comment (jlowdermilk) [inline .travis.yml]: `hack/verify-gendocs.sh` is already being run under the `script:` section. Fine to move it here, but delete from line 24 so we don't run it twice.

**Comment by jlowdermilk** [comment]
Not the first time this annoyance has happened. #5815 updated the pflags library which corrected how flag help is displayed. Unfortunately our build script is not smart about picking up on godep changes, so you'll need to run `make clean` and rebuild.

**Comment by thockin** [comment]
If it runs as part of travis, why is it not blocking PRs that needs docs
regenerated?

**Comment by thockin** [comment]
flipping make clean fixes it

**Pull request closed without merging**